portfolio/src/main/webapp/index.html

```diff
@@ -180,10 +180,13 @@ <h1>My Portfolio - Saakshi Saxena</h1>
                         </p>
                     </div>          
             </div>
-            <div id="comments-container"></div>
+            <div id="comments-div"><br>
+                <p style='font-size: larger;'><strong><em> Comments: </em></strong></p>
```

Comment:
Instead of using an inline style, and two tags, use a class and css-styling. HTML inline styles should be used rarely and mostly for properties which change programmatically.

Also: https://www.khanacademy.org/computer-programming/html-the-strong-and-em-tags/5341594157056000

Reply:
Ok, I'll do that.